technique/sans/instrument.py

```diff
@@ -728,11 +884,90 @@ def get_pv(self, name):
         return gen.get_pv(self._PV_BASE + name)
 
     def send_pv(self, name, value):
-        """Set the given PV within the sub heirarchy of the instrument.
+        """Set the given PV within the sub hierarchy of the instrument.
 
         For example, on Larmor, send_pv("DAE:WIRING_FILE", f) would
         change the value of the PV for "IN:LARMOR:DAE:WIRING_FILE" to
         the value in f.
-
         """
         return gen.set_pv(self._PV_BASE + name, value)
+
+    @property
+    def tables_path(self):
+        """Path to folder containing the detector/wiring/spectra
+        tables. The default path is normally:
+        "C:/Instrument/Settings/config/<instrument>/configurations/tables"
+        """
+        return self._tables_path
+
+    @tables_path.setter
+    def tables_path(self, new_path):
+        self._tables_path = r"" + new_path
```

Comment:
This doesn't look like it does anything

```suggestion
        self._tables_path = new_path
```